packages/SwingSet/src/types-external.js

```diff
@@ -177,6 +177,9 @@ export {};
  * @typedef {object} SwingSetOptions
  * @property {string} [bootstrap]
  * @property {ConfigProposal[]} [coreProposals]
+ * @property {string[]} [clearStorageSubtrees] chain storage paths identifying roots of subtrees
+ *   for which data should be deleted (except for overlaps with exportStorageSubtrees, which
+ *   are preserved).
```

Comment:
Why is this here?  Things that are explicitly chain related do not belong in SwingSet.  These three options (`coreProposals`, `clearStorageSubtrees`, and `exportStorageSubtrees`) should not be in this package anywhere.

Reply:
I think we've got some bleedover/(ab)use of the SwingSet `config` object and the thing of the same name in cosmic-swingset that includes additional non-SwingSet properties. `coreProposals` was the first, and these other two are following the same pattern, but I agree: cosmic-swingset should have a separate type that includes all three, so they won't be in the swingset package.

Reply:
Hmm, that's a good point. Updated.